streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBWindowStoreTest.java

```diff
@@ -573,8 +839,203 @@ public void testRolling() {
 
     }
 
+    private void verifyStoreContent1(final long startTime,
```

Comment:
Same question about this and the following methods... It looks like they're only used once. Should we inline them?

Reply:
I'm a bit skeptical for functions whose name need a number suffix to distinguish with others, since it usually indicates loss of generality :P Could the common logic be shared, or if not I'd suggest we still inline all of them.

Reply:
See comment above. The methods have nothing in common besides that they are verifying the store content at some point during the test. Not happy with the naming either, but had no better idea.